components/src/themed/ThemedSnackbar.tsx

```diff
@@ -16,8 +16,8 @@ export const ThemedSnackbar: React.FC<ThemedSnackbarProps> = (props) => {
     const currentTheme = useTheme(themeOverride);
     const theme = useAlternateTheme(
         themeOverride,
-        { colors: { accent: currentTheme.colors.textPalette.highlight } },
-        { colors: { accent: currentTheme.colors.primaryBase } }
+        { colors: { accent: currentTheme.colors.overrides.snackbar?.accent } },
+        { colors: { accent: currentTheme.colors.primaryPalette.dark } }
```

Comment:
Same question, should we add a fallback value to something that's guaranteed to be in a RNP theme?